functional-samples/tutorial.open-api-reference/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "manifest_version": 3,
+  "name": "Open extension API docs",
+  "version": "1.0.0",
+  "icons": {
+    "16": "icon-16.png",
+    "128": "icon-128.png"
+  },
+  "background": {
+    "service_worker": "sw.js",
```

Comment:
suggestion: rename to service-worker.js for folks who're not aware of the sw abbreviation